libbeat/beat/beat.go

```diff
@@ -149,6 +151,7 @@ func (b *Beat) Start() error {
 	// Additional command line args are used to overwrite config options
 	err, exit := b.CommandLineSetup()
 	if err != nil {
+		logp.Err("fails to load command line setup")
```

Comment:
These error messages somehow remain after rebasing with master. I am fixing them manually. Thanks for noticing it.